### ISSUE TITLE
Failing test: TestDockerFlag on Windows Docker

### ISSUE THREAD

**Issue by sharifelgamal** [kubernetes/minikube#10624]
https://storage.googleapis.com/minikube-builds/logs/10603/fb9f08b/Docker_Windows.html#fail_TestDockerFlags

**Comment by hetong07** [comment]
Found this info. Is it related?
```
I0225 21:04:41.619755   13864 out.go:129] * Using the docker driver based on user configuration
	I0225 21:04:41.620783   13864 start.go:276] selected driver: docker
	I0225 21:04:41.620783   13864 start.go:718] validating driver "docker" against <nil>
	I0225 21:04:41.620783   13864 start.go:729] status for docker: {Installed:true Healthy:true Running:false NeedsImprovement:false Error:<nil> Reason: Fix: Doc:}
	I0225 21:04:42.757367   13864 out.go:129] 
	W0225 21:04:42.757600   13864 out.go:191] X Docker Desktop only has 20001MiB available, you may encounter application deployment failures.
	X Docker Desktop only has 20001MiB available, you may encounter application deployment failures.
	W0225 21:04:42.758493   13864 out.go:191] * Suggestion: 
	
	    1. Open the "Docker Desktop" menu by clicking the Docker icon in the system tray
	    2. Click "Settings"
	    3. Click "Resources"
	    4. Increase "Memory" slider bar to 2.25 GB or higher
	    5. Click "Apply & Restart"
	* Suggestion: 
	
	    1. Open the "Docker Desktop" menu by clicking the Docker icon in the system tray
	    2. Click "Settings"
	    3. Click "Resources"
	    4. Increase "Memory" slider bar to 2.25 GB or higher
	    5. Click "Apply & Restart"
	W0225 21:04:42.758493   13864 out.go:191] * Documentation: https://docs.docker.com/docker-for-windows/#resources
	* Documentation: https://docs.docker.com/docker-for-windows/#resources
	I0225 21:04:42.762438   13864 out.go:129] 
	I0225 21:04:42.778343   13864 cli_runner.go:115] Run: docker system info --format "{{json .}}"
	I0225 21:04:43.968123   13864 cli_runner.go:168] Completed: docker system info --format "{{json .}}": (1.189779s)
```

**Comment by spowelljr** [comment]
In-progress

**Comment by spowelljr** [comment]
Potentially fixed by #11583; waiting for more runs to be sure.

**Comment by spowelljr** [comment]
Cause is:
```
E0609 05:39:26.230982   56824 status.go:413] kubeconfig endpoint: extract IP: "docker-flags-20210609051443-57368" does not appear in C:\Users\jenkins\minikube-integration\f78f6afe161a9d3fa4e95f6022c8d39c66c3f411\kubeconfig
```

**Comment by spowelljr** [comment]
Fixed with #11804
